src/protocol/logpoint.ts

```diff
@@ -352,31 +382,52 @@ export function setLogpointByURL(
 }
 
 const eventTypePoints: Record<string, PointDescription[]> = {};
+const eventTypeResults: Record<string, AnalysisEntry[]> = {};
 const eventTypeLogGroupId: Record<string, string> = {};
 
+// -jvv
 export async function fetchEventTypePoints(eventTypes: EventId[]) {
+  const mapper = eventLogpointMapper(/* getFrameworkListeners */ true);
   const sessionId = await ThreadFront.waitForSession();
 
   await Promise.all(
     eventTypes.map(async eventType => {
       const collectedPoints: PointDescription[] = [];
+      const collectedResults: AnalysisEntry[] = [];
       await analysisManager.runAnalysis(
         {
           sessionId,
-          mapper: `return [{ key: input.point, value: input }];`,
+          mapper,
           effectful: false,
           eventHandlerEntryPoints: [{ eventType }],
         },
         {
-          onAnalysisPoints: points => collectedPoints.push(...points),
+          onAnalysisPoints: points => {
+            // console.log("onAnalysisPoints", { points });
+            collectedPoints.push(...points);
+          },
+          onAnalysisResult: result => {
+            console.log("onAnalysisResult", { result });
+
+            const logGroupId = newLogGroupId();
+            eventTypeLogGroupId[eventType] = logGroupId;
+
+            console.log(">>>start waiting");
+            collectedResults.push(...result);
+            // const rv = await getUnwrappedFrameworkPoint(logGroupId, result);
```

Comment:
If there is a way to simplify this PR, it would probably have something to do with changing this `onAnalysisResult`. We could just immediately unwrap the result ("event") here and do something with it, instead of having to handle that separately within the action (`getEventListenerBreakpointTypes`)